app/views/layouts/admin.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Cwiczenie</title>
+  <%= stylesheet_link_tag    'admin', media: 'all' %>
+  <%= javascript_include_tag 'admin' %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+<%= render partial: 'admin/navbar' %>
+<% flash.each do |status, msg| %>
```

Comment:
you have the same piece of code on two different layouts. You can move it to the separate partial in shared/ folder